### DISCUSSION TITLE
Rainpoint integration into homeassistant

### DISCUSSION THREAD

**Discussion by sanvayquan** [community/community#142120] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Would you please give me the video guide how to integrate rainpoint into homeassistant ? I have followed you guide to integrate, but fail . thanks so much !

**Comment by LiteBrite82** [comment]
Hi @sanvayquan ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
